ckcp/tekton-results/openshift/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Please move `ckcp/tekton-results` to `ckcp/manifests/tekton-results`

Reply:
Moved.